test/test_catalog_api.py

```diff
@@ -306,11 +78,10 @@ def test_batch_delete_catalog_objects(self):
         self.assertIn(small_tea_id, response.deleted_object_ids)
 
     def test_batch_retrieve_catalog_objects(self):
-        ids = self.objects_by_client_id
-        coffee_id = ids[self.CLIENT_ID_COFFEE]
-        sales_tax_id = ids[self.CLIENT_ID_SALES_TAX]
-        beverages_id = ids[self.CLIENT_ID_BEVERAGES]
-        milks_id = ids[self.CLIENT_ID_MILKS]
+        coffee_id = self.catalog_fixture.object_id(self.catalog_fixture.CLIENT_ID_COFFEE)
```

Comment:
could you have a getter method on `CatalogFixture` to retrieve each object id such as: `self.catalog_fixture.coffee_object_id`